patches/minecraft/net/minecraft/client/model/geom/ModelLayers.java.patch

```diff
@@ -0,0 +1,12 @@
+--- a/net/minecraft/client/model/geom/ModelLayers.java
++++ b/net/minecraft/client/model/geom/ModelLayers.java
+@@ -199,7 +_,8 @@
+    }
+ 
+    public static ModelLayerLocation m_171291_(WoodType p_171292_) {
+-      return m_171300_("sign/" + p_171292_.m_61846_(), "main");
++      ResourceLocation location = new ResourceLocation(p_171292_.m_61846_());
++      return new ModelLayerLocation(new ResourceLocation(location.m_135827_(), "entity/signs/" + location.m_135815_()), "main");
```

Comment:
So I took a better look at this now as a reviewer. The solution in issue in #8131 while although valid is for the wrong reason. The error being displayed is a result of the `WoodType` having a `:` in the string passed into the path. Normally this wouldn't be an issue as vanilla would parse out the `:`. However, the additional prefix and delegation make the path invalid. If the entire string was considered, the namespace would instead error from `/`.

The `ModelLayerLocation` just represents a unique name for the root model. As such, this could have just replaced all `:` with `_` and worked as well. However, removing the hardcoded instance is always a better idea. As such, the initial prefix should be instead `sign/` instead of `entity/signs/` as that is indicative of the texture location and changes what the model name is otherwise.

Reply:
Done. Tests still work as well. Even loaded the first world I used.